shared/js/background/tab-manager.es6.js

```diff
@@ -116,6 +116,7 @@ class TabManager {
         let tab = tabManager.get({tabId: request.tabId})
 
         if (tab) {
+            tab.initiator = request.initiator
```

Comment:
initiator is the url of the previous page?

Reply:
I updated this to attach referrer directly to the tab - passing it around isn't perfect, and the initiator was not always what I wanted here.